lua/nvim-web-devicons/icons-default.lua

```diff
@@ -1286,6 +1304,18 @@ local icons_by_file_extension = {
     cterm_color = "166",
     name = "Mli",
   },
+  ["m"] = {
+    icon = "",
+    color = "#599eff",
+    cterm_color = "111",
+    name = "Tweak",
```

Comment:
Let's have some names please.

You can run `:NvimWebDeviconsHiTest` to preview them all.

Reply:
ok will check them out
btw tweaks are written with obj-c/obj-c++ instead of c/c++
but there aren't an obj-c/++ icon yet
so i counted them as c/c++

do i really have to use the names from `:NvimWebDeviconsHiTest`?

Reply:
Antidote is a zsh plugin manager

Reply:
> btw tweaks are written with obj-c/obj-c++ instead of c/c++

Ah I see, I thought "Tweaks" was a placeholder. Is tweaks some sort of programming language extension?

Either way, `m` and `mm` are the standard objective c and c++ extensions. Let's use generic names "ObjectiveC" and "ObjectiveCPlusPlus" to match the c++ names.

Reply:
> do i really have to use the names from `:NvimWebDeviconsHiTest`?

No, that's just useful to preview your changes alongside the existing.

Reply:
> > btw tweaks are written with obj-c/obj-c++ instead of c/c++
> 
> Ah I see, I thought "Tweaks" was a placeholder. Is tweaks some sort of programming language extension?
> 
> Either way, `m` and `mm` are the standard objective c and c++ extensions. Let's use generic names "ObjectiveC" and "ObjectiveCPlusPlus" to match the c++ names.

checkout r/jailbreak tweaks are used on jailbroken ios and ipasos